src/index.html

```diff
@@ -22,6 +22,483 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a
+          href="http://localhost:8080/"
+          class="header__logo-link"
+        >
+          <img
+            src="./images/logo.png"
+            alt="Moyo_shop_Gadgets"
+            class="header__logo-img"
+            width="40px"
+            height="40px"
+          />
+        </a>
+      </div>
+      <nav
+        class="header__nav"
+        aria-label="Main navigation"
+      >
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
```

Comment:
Consider replacing `href="#"` with a more meaningful link or a JavaScript function to prevent default behavior. Using `#` can cause the page to jump to the top when clicked, which might not be the desired behavior.